libdnf/dnf-transaction.cpp

```diff
@@ -1435,6 +1437,9 @@ dnf_transaction_commit(DnfTransaction *transaction, HyGoal goal, DnfState *state
             goto out;
     }
 
+    if (auto moduconteiner = dnf_sack_get_module_container(dnf_context_get_sack(priv->context)))
```

Comment:
Nitpick, but modueconteiner being misspelled is probably a future maintainability issue. I'd suggest:
```suggestion
    if (auto module_container = dnf_sack_get_module_container(dnf_context_get_sack(priv->context)))
```